cumulus/parachains/integration-tests/emulated/tests/assets/asset-hub-westend/src/tests/teleport.rs

```diff
@@ -89,13 +90,13 @@ fn penpal_to_ah_foreign_assets_receiver_assertions(t: ParaToSystemParaTest) {
 				pallet_balances::Event::Burned { who, amount }
 			) => {
 				who: *who == sov_penpal_on_ahr.clone().into(),
-				amount: *amount == t.args.amount,
+				amount: *amount >= fee_asset_amount / 2,
```

Comment:
These tests are kind of tricky to fix, because they set the fee amount to `fee_asset_amount / 2` before `t: ParaToSystemParaTest` is set, and `t: ParaToSystemParaTest` is not updated to reflect the changed fee. So then the amount check fails.